src/test/java/com/training/assignment/tests/AssignmentTest.java

```diff
@@ -0,0 +1,72 @@
+package com.training.assignment.tests;
+
+import com.training.assignments.Assignment;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AssignmentTest extends BaseTest{
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+    Assignment assignment = new Assignment();
+
+    @Test
+    public void testDataConversion() {
+        logger.info("double from int->" + assignment.getDoubleFromInt(55));
+        softAssert.assertThat(assignment.getDoubleFromInt(55))
+                .as("int number not converted to double").isEqualTo(55.0);
+        logger.info("int from double->" + assignment.getIntFromDouble(55.0));
+        softAssert.assertThat(assignment.getIntFromDouble(55.0))
+                .as("double number not converted to int").isEqualTo(55);
+        logger.info("float from double->" + assignment.getFloatFromDouble(55.0555558888888888888));
+        softAssert.assertThat(assignment.getFloatFromDouble(55.0555558888888888888))
+                .as("double number not converted to float").isEqualTo(55.055557f);
+        logger.info("double from float->" + assignment.getDoubleFromFloat((float) 55.00589001231313450));
+        softAssert.assertThat(assignment.getDoubleFromFloat((float) 55.00589001231313450))
+                .as("float number not converted to double").isEqualTo(55.005889892578125);
+        logger.info("byte from long->" + assignment.getByteFromLong(11005987L));
+        softAssert.assertThat(assignment.getByteFromLong(11005987L))
+                .as("long number not converted to byte").isEqualTo((byte) 35);
+        logger.info("long from byte->" + assignment.getLongFromByte((byte) 32));
+        softAssert.assertThat(assignment.getLongFromByte((byte) 32))
+                .as("byte number not converted to long").isEqualTo(32);
+    }
+
+    @Test
+    public void testOperationString() {
+        char[] getcharacter = assignment.getCharacterFromString("rccl");
+        for (char characters : getcharacter) {
```

Comment:
You can direclty call, assignment.getCharacterFromString("rccl"), in for each loop